src/datalab_app_plugin_insitu/nmr_insitu.py

```diff
@@ -35,27 +73,30 @@ def process_local_data(
         with tempfile.TemporaryDirectory() as tmpdir:
             if folder_name.endswith(".zip"):
                 with zipfile.ZipFile(folder_name, "r") as zip_ref:
-                    zip_ref.extractall(tmpdir)
+                    members = [
+                        m for m in zip_ref.namelist() if not ("__MACOSX" in m or m.startswith("."))
```

Comment:
Same comment as above, lets try to have these special values listed as a utility function that we can use on strings or paths